spec/features/admin_role_spec.rb

```diff
@@ -70,7 +72,14 @@
 
       # Edit details and add campaign
       click_button('Edit Details')
+
       page.all('.button.border.plus')[4].click
+
+      # Ensure campaigns appear in select list ordered by time (descending)
+      campaign_options = all('select[name=campaign]>option')[1,2]
```

Comment:
`all('select[name=campaign]>option')` returns an array of the options in that select element.

`array[1,2]`returns a portion of the array, starting from index 1, with length 2. https://docs.ruby-lang.org/en/2.3.0/Array.html#method-i-5B-5D